meta.template.js

```diff
@@ -7,7 +7,6 @@
 // @updateURL    [UPDATE_URL]
 // @homepageURL  https://adguard.com/
 // @author       AdGuard
-// @include https://www.google.*/*
```

Comment:
just to remove @include is not enough, because this script will not be executed at all.
it's better to replace it with `// @match *://*/*`
check https://wiki.greasespot.net/Include_and_exclude_rules

Reply:
Fixed

Reply:
Fixed https://github.com/AdguardTeam/DisableAMP/pull/19/commits/fa03b779f8975fdf512c7281b8432d32826e8d0c